genesis/engine/bvh.py

```diff
@@ -186,12 +195,7 @@ def build(self):
             for i in ti.static(range(3)):
                 self.scale[i_b][i] = ti.select(scale[i] > 1e-7, 1.0 / scale[i], 1)
```

Comment:
I would recommend this instead: `self.scale[i_b][i] = ti.select(scale[i] > gs.EPS, 1.0 / scale[i], 1.0)`